src/rockset_sqlalchemy/client/cursor.py

```diff
@@ -74,6 +78,21 @@ def fetchone(self):
 
         return tuple(result)
 
+    def _response_to_column_fields(self, column_fields):
```

Comment:
pl do some code comments that specifies that we are looking at the result set to reverse-engineer the column types